autogpt_platform/backend/schema.prisma

```diff
@@ -186,6 +289,9 @@ model AgentNodeExecutionInputOutput {
   referencedByOutputExecId String?
   ReferencedByOutputExec   AgentNodeExecution? @relation("AgentNodeExecutionOutput", fields: [referencedByOutputExecId], references: [id], onDelete: Cascade)
 
+  configuredAgentId String?
+  ConfiguredAgent   ConfiguredAgent? @relation("AgentNodeExecutionInput", fields: [configuredAgentId], references: [id])
```

Comment:
Once an `AgentNodeExecutionInputOutput` has been created, it is time-invariant, so it should be tied to the corresponding agent *version*. Otherwise we can't interpret the inputs/outputs properly as they are tied to specific nodes in a specific graph version.

Reply:
Also, this references `@relation("AgentNodeExecutionInput"` which I don't think is correct (copy-pasted?)

Reply:
Yes it was, thanks for pointing it out. I'll check what the correct way to handle that is 👍